builtin/providers/aws/resource_aws_devicefarm_project.go

```diff
@@ -0,0 +1,111 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/devicefarm"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsDeviceFarmProject() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsDeviceFarmProjectCreate,
+		Read:   resourceAwsDeviceFarmProjectRead,
+		Update: resourceAwsDeviceFarmProjectUpdate,
+		Delete: resourceAwsDeviceFarmProjectDelete,
+
+		Schema: map[string]*schema.Schema{
+			"arn": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+		},
+	}
+}
+
+func resourceAwsDeviceFarmProjectCreate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).devicefarmconn
+	region := meta.(*AWSClient).region
+
+	//	We need to ensure that DeviceFarm is only being run against us-west-2
+	//	As this is the only place that AWS currently supports it
+	if region != "us-west-2" {
+		return fmt.Errorf("DeviceFarm can only be used with us-west-2. You are trying to use it on %s", region)
```

Comment:
The provider just hangs until timeout. It took me quite a while before it showed me this:

```
make testacc TEST=./builtin/providers/aws TESTARGS='-run=DeviceFarm' 2>~/tf.log
go generate ./...
TF_ACC=1 go test ./builtin/providers/aws -v -run=DeviceFarm -timeout 90m
=== RUN   TestAccAWSDeviceFarmProject_basic
--- FAIL: TestAccAWSDeviceFarmProject_basic (448.13s)
	testing.go:137: Step 0 error: Error applying: 1 error(s) occurred:

		* aws_devicefarm_project.foo: Error creating DeviceFarm Project: RequestError: send request failed
		caused by: Post https://devicefarm.eu-west-1.amazonaws.com/: dial tcp 81.200.64.50:443: i/o timeout
FAIL
exit status 1
FAIL	github.com/hashicorp/terraform/builtin/providers/aws	448.149s
```